src/table.rs

```diff
@@ -73,6 +73,7 @@ table! {
     b"GDEF" => opentype::GlyphDefinition(),
     b"GPOS" => opentype::GlyphPositioning(),
     b"GSUB" => opentype::GlyphSubstitution(),
+    b"MATH" => opentype::MathHeader(),
```

Comment:
I think it should be `Mathematics`, which will include `Header` and `Constants`.